### ISSUE TITLE
Carousel on Mobile View

### ISSUE THREAD

**Issue by Johanna-hub** [WageCheck/workerbird-website#17]
The carousel is great.  Well done.  If we are going to go ahead with carousel on mobile view (depends on if buttons are a bit small and fiddly but looks good to me to be honest), the layout should be:

(no icons, only for desktop)

Screen 1
Screenshot
Track your working life

Screen 2
Screenshot
Get insights into your working day

Screen 3
Screenshot
Access advice and support

**Comment by tspeed90** [comment]
To clarify @Johanna-hub, you are saying there should be no descriptive text about each feature on mobile. Just an image of the screenshot and the feature itself?

**Comment by Johanna-hub** [comment]
Yes, a screenshot - then the text I've written above as simple caption

**Comment by tspeed90** [comment]
Thanks! One more thing, the screenshot images you sent on gitter mostly work except for one. The Track screen image is cropped some strange way. Do you have another screenshot for that screen?

**Comment by Johanna-hub** [comment]
![screen shot 2018-11-07 at 12 57 36](https://user-images.githubusercontent.com/24194579/48132961-12e40680-e28d-11e8-87ee-04e8fe97765d.png)

**Comment by Johanna-hub** [comment]
Is that better?  Can we use that one as Screen 1
The graph one you have already for Screen 2 
And this one for Screen 3:
![screen shot 2018-11-07 at 12 58 42](https://user-images.githubusercontent.com/24194579/48132992-373fe300-e28d-11e8-8047-e64129609a1e.png)

**Comment by Johanna-hub** [comment]
![screen shot 2018-06-28 at 21 22 54](https://user-images.githubusercontent.com/24194579/48150304-2ce60f00-e2b6-11e8-9b4c-697da48c64dc.png)
